prime-router/metadata/schemas/ND/nd-covid-19.schema

```diff
@@ -9,117 +9,251 @@ elements:
     # msh section
     - name: sending_application
 
-    - name: reporting_facility_name
-
-    - name: reporting_facility_clia
-
     - name: receiving_application
       default: Maven^2.16.840.1.114222.4.3.4.34.1.1^ISO
 
     - name: receiving_facility
       default: NDDOH^2.16.840.1.113883.3.89.109.100.1.3^ISO
```

Comment:
Added schemas for each state so they have a default value for the `MSH` segment for `receiving_application` and `receiving_facility`.  I still think this should probably come from metadata, probably the Organization Service, since there's a one-to-one correlation, but for now it can stay here.